elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -24,6 +24,15 @@
 INDENT_OFFSET = 0
 TAB_SIZE_PX = 50
 
+DAG_FIRST_WRONG_FEEDBACK = [
```

Comment:
Can we make this a dict with descriptive keys? Maybe `feedback-none` and `feedback-first-wrong`?

Reply:
sure!